docs/community/tutorials-contest/2022/avalanche-theGraph-index/README.md

```diff
@@ -0,0 +1,381 @@
+
+
+2. Initialising a Default Subnet
+
+We will be using the `avalanche subnet create` command line wizard to get our network running. ASAP.
+In the same directory where the binary was installed, run the following command
+
+```zsh
+avalanche subnet create <SubnetName>
+```
+Substitute `<SubnetName>` with any perferred name of your choice but without spaces. For this tutorial we are going to call our Subnet `<fibrinNet>`.
+
+```zsh
+avalanche subnet create fibrinNet
+```
+
+Since this command does not provide any arguements, you would neeed to walk through the configuration wizard to generate a `genesis file` for your network.
+
+* Choose a Virtual Machine (VM): 
+  ![choose a VM](./images/2.png "Choose VM")
+  We are going to be making use of the `SubnetEVM`
+
+* Pick a chain ID
+  ![chain ID](./images/3.png "Chain ID")
+  Every EVM based blockchain has a special parameter called a `chainID`. ChainIDs are expected to be unique values, you can check [chainlist.org](https://chainlist.org/) to see if your proposed chain ID is already in use. We will be making use of the chain ID `1970` (A pun on JavaScript dates...lol).
+
+* Select a symbol for the native Subnet token
+  ![symbol](./images/4.png "token symbol")
+
+* Set fees: Select the `low disk use / low throughput` option
+  ![fees](./images/5.png "fees")
+
+* Airdrop: default to airdrop 1 million tokens to provided address
+  ![airdrop](./images/6.png "airdrop")
+
+* Add a custom precompile to modify the EVM: For this section, we will not be using a pre-compile script
+  ![precompile](./images/7.png "precompile")
+
+
+The wizard won't customize every aspect of the Subnet-EVM genesis for you, we will be doing this in the subsequent sections.
+
+![complete](./images/8.png "complete")
+
+To view the list of all created Subnets, just execute the following command
+
+```zsh
+avalanche subnet list
+```
+![list](./images/9.png "list")
+
+
+3. Deploying the Subnet Locally.
+
+To deploy the newly created Subnet locally, run the following command
+
+```zsh
+avalanche subnet deploy <SubnetName>
+```
+
+![deploy](./images/10.png "deploy")
+
+When a Subnet is run locally, it starts a multi-node (5 node) Avalanche Network in the background.
+
+![deploy_f](./images/11.png "deploy_f")
+
+To test the functionality of the just created Subnet, go ahead and add the configuration details to [Metamask](https://metamask.io/).
+You can create a new metamask account by importing the private key `0x56289e99c94b6912bfc12adc093c9b51124f0dc54ac7a766b2bc5ccf558d8027` and start experiencing with this account.
+
+I have a [Github Tutorial](https://github.com/FibrinLab/Avalanche-Local-Environment-Setup) that explains how to setup your local development environmet including `Metamask`. 
+
+Lastly don't forget to stop the running local network
+```zsh
+avalanche network stop <snapshotName>
+```
+
+![deploy_f](./images/12.png "deploy_f")
+
+
+
+### Deploying Smart Contracts with Remix
+
+1. Setting up Remix
```

Comment:
Could you make this (and other similar places) a heading (###), instead of an ordinal?

Reply:
This has been corrected